InfoMovies/InfoMovies/Views/ItemDetailsPage.xaml

```diff
@@ -0,0 +1,127 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Views:BasePage xmlns="http://xamarin.com/schemas/2014/forms"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                x:Class="InfoMovies.Views.ItemDetailsPage"
+                xmlns:Extensions="clr-namespace:InfoMovies.Extensions"
+                xmlns:Views="clr-namespace:InfoMovies.Views"
+                xmlns:ImageLoading="clr-namespace:FFImageLoading.Forms;assembly=FFImageLoading.Forms">
+    <ContentPage.Content>
+        <AbsoluteLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
+            <ScrollView Orientation="Vertical" Style="{StaticResource scrollViewStyle}" BackgroundColor="DarkGray"
+                        AbsoluteLayout.LayoutFlags="All" AbsoluteLayout.LayoutBounds="0,0,1,1">
+                <StackLayout Orientation="Vertical" Style="{StaticResource stackLayoutStyle}">
+                    <Grid Style="{StaticResource gridStyle}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="70"/>
+                            <RowDefinition Height="250"/>
+                            <RowDefinition Height="150"/>
+                        </Grid.RowDefinitions>
+
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="30*"/>
+                            <ColumnDefinition Width="70*"/>
+                        </Grid.ColumnDefinitions>
+
+                        <Views:GradientMask TopColor="Black" BottomColor="Gray" 
+                                        Grid.Row="0" Grid.Column="0" Grid.RowSpan="3" Grid.ColumnSpan="2"/>
+
+                        <ImageLoading:CachedImage HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"
+                                                  Grid.Row="0" Grid.RowSpan="3" Grid.Column="0" Aspect="AspectFit" Source="{Binding Path=Poster}"/>
+
+                        <StackLayout Orientation="Vertical" Style="{StaticResource stackLayoutStyle}"
```

Comment:
Do you really need this StackLayout? If so, why?

Reply:
no really...
my bad